devicon.json

```
@@ -8231,6 +8231,22 @@
         "color": "#ff7f7f",
         "aliases": []
     },
+    {
+        "name": "replit",
+        "altnames": [],
+        "tags": [
+            "software"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark"
+            ],
+            "font": []
+        },
+        "color": "#F26207",
+        "aliases": []
```

Since `original` only has a single color, and has a unified path it can also be used as a plain icon
```suggestion
            "font": [
                "original"
            ]
        },
        "color": "#F26207",
        "aliases": [
            {
                "base": "original,
                "alias": "plain"
            }
        ]
```

---

> Since `original` only has a single color, and has a unified path it can also be used as a plain icon

Yeah, I had thought about it but still I used `original`.